Functions/Functions.cs

```diff
@@ -0,0 +1,79 @@
+
+using Godot;
+using System;
+using System.Collections.Generic;
+
+namespace Functions {
+
+    /// <summary>
+    /// For Debugging purposes. If <c> Enabled = true </c>, debug output related <br/>
+    /// to timing and playbak status will be enabled.
+    /// </summary>
+    public static class Debugging {
+        public static readonly bool Enabled = false;
+    }
+
+    /// <summary>
+    /// Class for Sycnhronous Timing within the application. 
+    /// </summary>
+    public partial class TimeKeeper {};
+    
+    namespace Sonification {
```

Comment:
For organization projects usually have namespaces directly reflect the path to a file. So e.g. `Function` would be defined in `Functions/Sonification/Function.cs`, not `Functions/Function.cs`

Reply:
That makes sense. I put the `Functions/Sonification` in so I could keep everything that dealt with only sound generation in there and I left `Function `out because I figured those nodes could be used within the` Timeline UI` as well, but that seems to be bad reasoning now. Would you suggest we get rid of `Sonification` and have everything in `Functions`?

Reply:
I don't mind the sonification namespace, it's just organized weirdly; You shouldn't have two instances of the keyword `namespace` in one file.

Reply:
I'm down for a reorganization to make it not weird. I didn't know it was bad practice to have the `namespace` keyword occur more than once in a file, I was just going off of this: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/namespaces#:~:text=is%20semantically%20equivalent%20to. Does that mean I should have a new `.cs` file w/ `namespace Functions.Sonfication` in it instead of nesting them like it is right now?

Reply:
The usual rules are:
- Every class goes in its own .cs file with the same name
- Every class's directory path is the same as its namespace. So if the namespace of a class is `Functions.Sonification` then its path is `Functions/Sonification/Class.cs`
- Use the shortform namespace syntax (just a `namespace X.Y.Z;` at the top of the file after your `using`s)
- You basically never need partial classes. We *do* need them for godot stuff because godot generates more stuff to add into them, but for normal purposes classes are defined in one place.

and yeah probably make some new .cs files to split it up.

Reply:
Alright, thanks that makes sense. With the added context, I think I should probably leave the `Functions` namespace to what you've written for the Functions Folder (b/c like you said, namespaces correspond with and are folders), and probably just put everything I did in a new folder and namespace (Sonification I suppose). The last part explains why I got an error when initially trying to remove the partial declaration from the `TimeKeeper` class too, thanks.